db/table_cache.cc

```diff
@@ -629,6 +629,7 @@ size_t TableCache::GetMemoryUsageByTableReader(
   auto table_reader = fd.table_reader;
   // table already been pre-loaded?
   if (table_reader) {
+    assert(s.ok());
```

Comment:
Can you just move the declaration of s to after this check?

Reply:
sure